typewriter/fixes/fix_annotate_docs.py

```diff
@@ -152,14 +152,16 @@ def parse_docstring(self, docstring, line):
     def should_skip(self, node, results):
         return False
 
-    def type_updater(self, match):
-        # type: (Match) -> str
+    def type_updater(self, match, node):
+        # type: (Match, Node) -> str
         # Replace `pkg.mod.SomeClass` with `SomeClass`
         # and remember to import it.
         word = match.group()
         # Assume it's either builtin or from `typing`
         if word in typing_all:
-            self.add_import('typing', word)
+            if not does_tree_import('typing', word, node):
+                # Check whether import exists already
+                self.add_import('typing', word)
```

Comment:
I see this pattern repeated twice:

```python
        if word in typing_all:
            if not does_tree_import('typing', word, node):
                # Check whether import exists already
                self.add_import('typing', word)
```

I think it would be nice to wrap it up into a `add_typing_import(word, node)` method.

Reply:
i added `touch_typing_import` because I think that's more accurate but if you think it's misleading just let me know